### ISSUE TITLE
I am stuck on the frontpage

### ISSUE THREAD

**Issue by Ilyes512** [wardrobecms/wardrobe-archived#17]
How do I get to see the admin area and the login area. I got trough the install and now I get to see the frontpage with a option to either go to post (empty) or rss. 

So how to proceed from this? I tried /login without succes.

edit:
Do I need to go into routes and create it my self? Just started learning Laravel.

**Comment by ericlbarnes** [comment]
It should be /wardrobe for the admin. Can you give that a try?

**Comment by Ilyes512** [comment]
Tnx, I got it running :). It's indeed very minimalistic, but a great start! :)

Just got a few comments:
1) I had allot of trouble getting it installed cause of permission issues. I had to try all kind of things and I resolved to chmod -R 777 the hole wardrobe folder. I am running this with Vagrant (VirtualBox) and it probably got to do with that as well. I will try it again later with a fresh install to see if I can get it running without chmod'ing the whole folder.

2) I couldn't find out on my own where to login until you told me :). Maybe a redirect to the /wardrobe after the install is complete and a mentioning of /wardrobe in the documentation/github might be handy?

edit:
I did the setup again in a fresh vagrant box and I still get permission issue's. I chmoded app/storage (with vagrant and within my real os) multiple times. I now get the fallowing error:
![image](https://f.cloud.github.com/assets/2445415/1467050/58c46636-457b-11e3-9073-1217ce8f08bb.png)

So there might be some files being copied while installing thats needs permissions?

edit2:
I see that new files created inside the storage folder do not have the same permission as the folder: -rw------- (as shown by "ls -l" command). So its a issue with vagrant... hmm

**Comment by Ilyes512** [comment]
Ok I just fixed it :) by chaning my vagrant file:

```
config.vm.synced_folder "~/WebDev", "/var/www/", id: "vagrant-root", :nfs => false, owner: "www-data", group: "www-data"
```

Just 2 things:
- I found out that you already show a link to the login area after the install is done, however I didn't know it was a link until I hovered over it.
- The documentation says the fallowing: `chmod 777 public/images`. But there is no folder `images`, there is a however a folder called `img`. So there is a little confusion to what needs to be done. Do I create a new folder `images` or did you mean `img` in the first place?
